ocrd_lib.sh

```diff
@@ -245,85 +247,77 @@ post_process_to_mets() {
   # perhaps if URL_PREFIX:  mm-update -m "$METS_PATH" validate -u $URL_PREFIX
 }
 
-kitodo_production_task_action() {
-  ACTION=""
+# exit in async or sync mode
+close() {
+  if test "$ASYNC" = true -o "$ASYNC" = "true"; then
+    logger -p user.info -t $TASK "ocr_exit in async mode - immediate termination of the script"
+    # prevent any RETVAL from being written yet
+    trap - EXIT
+    exit 1
+  else
+    # become synchronous again
+    logger -p user.info -t $TASK "ocr_exit in sync mode - wait until the processing is completed"
+    wait $!
+  fi
+}
+
+webhook_send() {
```

Comment:
I suggest we use the chance to rename the first arg from a number to the event name itself, which is self-documenting.

Reply:
I've made changes. While there were functions before, now it's even clearer."

Reply:
Thanks!